nbi/engine/build.xml

```diff
@@ -141,7 +141,7 @@
     </target>
 
     <target name="probe">
-        <javac srcdir="probesrc" destdir="${build.classes.dir}/org/netbeans/installer/utils/applications/" source="1.3" debug="true" deprecation="true" target="1.1"/>
+        <javac srcdir="probesrc" destdir="${build.classes.dir}/org/netbeans/installer/utils/applications/" source="1.6" debug="true" deprecation="true" target="1.6"/>
```

Comment:
This is tricky. I guess I was hoping we'd find a way to still compile with -target 1.1 here, as when compiling with -target 1.6, we wouldn't be able to detect JDKs < 6 (not *that* bit deal, but would be nice if we had a way to detect the JDK and say it is not supported, as opposed to failing the detection).